index.html

```diff
@@ -329,21 +329,30 @@ <h2><dfn>ParityType</dfn> enum</h2>
 This section defines the <dfn>Security considerations</dfn> for the <cite>
 Serial API</cite>.
 
-It is RECOMMENDED that an user interface be presented to the user to allow them
-to select the serial ports that the page can access. Such an interface would
-allow an end user to select zero ore more serial ports on the device - or to
-reject the request to access any ports.
+It is RECOMMENDED that a user interface be presented to the user to allow them
+to select the serial ports that the page can access. Such an interface SHOULD
+allow an end user to select a serial port on the device or to reject the request
```

Comment:
Maybe "decline"... sounds less "promisey" than "reject" :) 

```suggestion
allow an end user to select a serial port on the device or to decline the request
```